src/app/components/challenge/challengeparticipate/challengeparticipate.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="challenge-card">
-  <div *ngIf="!isActive" class="ev-card-panel ev-md-container card-bt-margin">
+  <div *ngIf="!isActive && isLoggedIn" class="ev-card-panel ev-md-container card-bt-margin">
```

Comment:
@robingan7 By adding this, if challenge is not active then the message `Sorry, the challenge is not active` will not show if user is not logged in which is not the expected behaviour.

Reply:
@Sanji515 Is it suppose to show `Please Login to access this section. ` no matter the challenge is active or not when the user is not logged in

Reply:
> Is it suppose to show Please Login to access this section. no matter the challenge is active or not when the user is not logged in

Yes, if this is the case then putting `isLoggedIn` condition is correct. In the current version it check for only if the challenge is active or not and shows the message `Sorry, the challenge is not active` which I think should not happen. The flow should be like first user will login and then see the challenge participation status. So I think putting `isLoggedIn` condition is correct :+1: